FinancialTransactionsApi/V1/Gateways/PostgreDbGateway.cs

```diff
@@ -32,7 +32,11 @@ public async Task<IEnumerable<Transaction>> GetByTargetId(string targetType, Gui
             return response?.ToDomain();
         }
 
-        public Task<Transaction> GetTransactionByIdAsync(Guid targetId, Guid id) => throw new NotImplementedException();
+        public async Task<Transaction> GetTransactionByIdAsync(Guid id)
+        {
+            var data = await _databaseContext.TransactionEntities.AsNoTracking().Where(t => t.Id == id).FirstOrDefaultAsync().ConfigureAwait(false);
```

Comment:
can you reformat this line so that it is more readable like this:

```
var data = await _databaseContext.TransactionEntities
                                     .AsNoTracking()
                                     .Where(t => t.Id == id)
                                     .FirstOrDefaultAsync()
                                     .ConfigureAwait(false);
```